weed/server/filer_server_handlers_read.go

```diff
@@ -119,7 +119,16 @@ func (fs *FilerServer) GetOrHeadHandler(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	if r.URL.Query().Has("metadata") {
+	query := r.URL.Query()
+	if query.Has("metadata") {
```

Comment:
Please be consistent and use "metadata=true". Please also fix the wiki.

Reply:
> Please be consistent and use "metadata=true". Please also fix the wiki.

@chrislusf done